### ISSUE TITLE
"-" in @Scheduled's cron expression causes "invalid char(s) in expression" error

### ISSUE THREAD

**Issue by fbricon** [redhat-developer/intellij-quarkus#1206]
The "-" in the cron expression causes an "invalid char(s) in expression" error. The value exists in application.properties, and navigation works fine
```java
@Scheduled(cron = "{marketplace-api.crawl.cron}")
public void crawl() {
}
```
<img width="844" alt="Screenshot 2023-10-06 at 11 10 39" src="https://github.com/redhat-developer/intellij-quarkus/assets/148698/5a574915-ef5c-4e62-a494-19cecb510508">

**Comment by angelozerr** [comment]
@fbricon is this issue has been fixed with your last PR?

**Comment by fbricon** [comment]
Fixed with #1326
